sdk/resourcemanager/azure-resourcemanager-appplatform/pom.xml

```diff
@@ -83,9 +83,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-identity</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager-test</artifactId>
+      <version>2.0.0-beta.4</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-test;current} -->
```

Comment:
I see that there is apache-commons dependency included. This dependency is not allowed. Can this dependency be removed?

@JonathanGiles

Reply:
I think this apache-commons is required for spring cloud service(named appplatform here). It could help on compression and upload.

cc: @ChenTanyi

Reply:
Is there any compression package can be used?

Reply:
Yes - we should start with the default position that this dependency is not allowed, and work from there to either find an alternative approach or justify why this is needed, but for me the hurdle for accepting this dependency is very high.